apps/challenges/models.py

```diff
@@ -124,7 +124,7 @@ def __init__(self, *args, **kwargs):
     class Meta:
         app_label = 'challenges'
         db_table = 'challenge_phase'
-        unique_together = (('codename', 'challenge'),)
+        unique_together = (('codename', 'challenge'), ('id', 'codename'),)
```

Comment:
We don't need this change.

Reply:
So, we need the additional `unique_together` here with attributes `pk` and `dataset_split`